Client/Frontend/Browser/Tabs/InactiveTabItemCell.swift

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class InactiveTabItemCell: UITableViewCell, NotificationThemeable, ReusableCell {
```

Comment:
Should this be a more generic name? Do we wanna reuse this new dynamic type enabled cell elsewhere?

Reply:
I was thinking about creating a more generic cell however there are 5 different variations of the old OneLineTableViewCell and when I tried to combine them all using one model the code became quite messy. I think it would make more sense to split OneLineTableViewCell into 2-3 versions that have clearer code.

Reply:
That make sense to me, the challenge will be to find the right name to describe them 😂